src/client.js

```diff
@@ -31,12 +37,16 @@ client
   .on('error', (err) => client.logger.error(err))
   .on('warn', (warn) => client.logger.warn(warn));
 
-// Login client
+/**
+ * Login client with BOT_TOKEN from .env
+ */
 if (ENV !== envs.TESTING) {
   client.login(BOT_TOKEN);
 }
 
-// Add message listener
+/**
+ * Listen for message and send to dispatch when one is received
+ */
 client.on('message', (message) => dispatchCommand(message));
 
 export default client;
```

Comment:
This should have a docstring while everything above have comments. Docstrings are used for variables and functions (especially exports), while the internal logic doesn't need that.

Reply:
Good catch, I wasn't checking what each one was on